docs/misc_resources/defining_filepaths.md

```diff
@@ -0,0 +1,54 @@
+# Specifying Filepaths in H2I
+
+Filepaths in H2I can be specified in a number of ways. Files are looked for using the `find_file()` method, which looks for files in the following order:
+1. With respect to a specified root folder
+2. With respect to the current working directory (cwd)
+3. With respect to the H2Integrate package root. This folder includes subdirectories of `examples`, `resource_files`, `docs` and `h2integrate`.
+4. As an absolute filepath
+
+## Main Input Files
+The main input files to H2Integrate (`plant_config`, `tech_config` and `driver_config`) are defined in a top-level config file. In many examples, these files live in the same folder as the python run script.
+
+The main input files are searched for in H2I in the following order:
+1. Relative to the folder containing the top-level config file
+    ```yaml
+    plant_config: "plant_config.yaml"
+    technology_config: "tech_config.yaml"
+    driver_config: "driver_config.yaml"
+    ```
+2. Relative to the current working directory (suppose our cwd is `examples/14_wind_hydrogen_dispatch`):
+    ```yaml
+    plant_config: "inputs/plant_config.yaml"
+    technology_config: "inputs/tech_config.yaml"
+    driver_config: "inputs/driver_config.yaml"
+    ```
+3. Relative to the H2Integrate root directory
+    ```yaml
+    plant_config: "examples/01_onshore_steel_mn/plant_config.yaml"
+    technology_config: "examples/01_onshore_steel_mn/tech_config.yaml"
+    driver_config: "examples/01_onshore_steel_mn/driver_config.yaml"
+    ```
+4. As an absolute filepath.
+    ```yaml
+    plant_config: "/Users/myname/H2Integrate/examples/01_onshore_steel_mn/plant_config.yaml"
+    technology_config: "/Users/myname/H2Integrate/examples/01_onshore_steel_mn/tech_config.yaml"
+    driver_config: "/Users/myname/H2Integrate/examples/01_onshore_steel_mn/driver_config.yaml"
+    ```
+
+Note that if more than one file is found with the same name, then the code will give an error. A filepath will only be returned if there is one, and only one, file that matches.
```

Comment:
If I'm reading this note correctly, if H2I successfully finds a file using e.g. methods 1 and 2 above, then an error is raised? I looked at the actual code and don't think I saw that logic, so I think this doc could be more clearly worded. Is this only if there's a filename overlap within _one_ of the given search methods?